src/main/target/STM32G47X/target.h

```diff
@@ -54,9 +54,7 @@
 
 #define USE_BEEPER
 
-#ifdef USE_SDCARD
-#define USE_SDCARD_SPI
-#endif
+#undef USE_SDCARD
```

Comment:
🤔 should be part of common_post.h ?

Reply:
I need to work through this one a little more. G4 seems to require a sdio_g4xx.c file that does not exist. So I am simply UNDEF it here to ensure it is not a failure. Further work on the G4 target required.

Reply:
Also I will be adding instructions on what should / should not be included in PRE and POST. I think those files are a bit of a mess at the moment, but thats largely due to the gating. I plan on cleaning up the gating for 4.5.